source/content/partials/traffic-limits-overages.md

```diff
@@ -12,18 +12,18 @@ reviewed: ""
 
 Pantheon monitors your site traffic as part of our evaluation of overall site health. To understand the limits associated with your pricing plan, visit the [pricing comparison page](https://pantheon.io/plans/pricing/pantheon-web-hosting-pricing-comparison?docs) for additional information.
 
 ### Overage Protection
 
-Pantheon designed Overage Protection for Performance sites to prevent one-time traffic spikes from causing billing issues. All Performance plans and higher include Overage Protection, which provides billing protection against externally driven spikes, or for businesses that have an annual “big event” but otherwise operate at a lower “normal” rate.
+Pantheon designed Overage Protection for Performance and Elite sites on annual plans to prevent one-time traffic spikes from causing billing issues. All Performance plans and higher (on an annual agreement) include one month of Overage Protection, which provides billing protection against externally driven spikes, or for businesses that have an annual “big event” but otherwise operate at a lower “normal” rate. This protection resets with the calendar year and will be automatically applied the first month of the year your site goes over its plan limits. You as the site owner do not need to do a thing. In a month where Overage protection is applied the site owner will be notified of the event and no other action will be taken. This notice is a great time to review your site's traffic to determine if it is time to move to a new plan and avoid the risk of a future overages bill. Qualified nonprofits receive additional overage protection at no extra cost, automatically applied. Refer to the [FAQ](https://docs.google.com/document/d/1vaLJHXJJashwO9zsHFeywW-RQx_6uWOWH2Yr31sbQv4/edit?usp=sharing) to learn more about how to qualify for these and other benefits available only to Pantheon's nonprofit customers.
 
-If the change to traffic behavior exceeds your plan limit for any two months of traffic, your site will be moved to the next appropriate plan to help avoid further overages. You will receive notifications of this change in an email with a subject of "You've been right-sized" ahead of time.
+If the change to traffic behavior exceeds your plan limit for a second month in any calendar year, you will receive notice of the traffic volume and the impact of the excess traffic. For online customers (those not on a contract) that will result in an invoice later that same month. For customers on contract overages will be billed quarterly. Refer to the [FAQ](https://docs.google.com/document/d/1vaLJHXJJashwO9zsHFeywW-RQx_6uWOWH2Yr31sbQv4/edit?usp=sharing) for specifics on what to expect when you receive a notification that your site is over its plan limits.
```

Comment:
@IngridKwok, is "online customers" a phrase we use externally? I think it would be more direct and clearer here to say "customers on a monthly plan". @carly-armstrong Would that phrase be accurate?

Reply:
The lines below use the phrase
> Sites on a monthly agreement (Basic and Performance)

Reply:
@stevector @carly-armstrong Credit card customers and contract customers are the terms used in the FAQ.  Also annual subscriptions and monthly subscriptions.